back/boxtribute_server/business_logic/statistics/sql.py

```diff
@@ -165,10 +156,43 @@
         h.size_id
     FROM HistoryReconstruction h
     WHERE h.changes = 'box_state_id'
+),
+CreatedDonatedBoxes AS (
+    SELECT
+        h.box_id,
+        h.box_state_id,
+        date(h.changedate) as moved_on,
+        h.items AS number_of_items,
+        h.location_id,
+        h.product_id AS product,
+        h.size_id
+    FROM HistoryReconstruction h
+    WHERE h.changes = 'Record created' and h.box_state_id = 5
 )
 
--- Main query to perform aggregation and grouping of boxes being moved between states
--- InStock and Donated
+-- MAIN QUERY
+
+-- Collect information about boxes created in donated state
```

Comment:
@vahidbazzaz do you mind reviewing this query incl. the CTE `CreatedDonatedBoxes` above?
Its goal is to obtain statistics of boxes created with state "Donated" (=5).